src/components/TrackTile/TrackerDetails/TrackAmountControl.tsx

```diff
@@ -87,7 +107,12 @@ const TrackAmountControl: FC<Props> = ({ color, value, onChange }) => {
           'Increment tracker value',
         )}
         accessibilityRole="button"
-        onPress={() => onChange(value + 1)}
+        onPress={() => {
+          const asString = addToNumberString(currentValue, 1, false) as string;
```

Comment:
Using currentValue rather than an unstable incoming value ensures that we are consistently incrementing the "saved" value rather than it getting trampled when the upsert query resolves.